README.md

```diff
@@ -11,7 +11,7 @@ Many bacterial and archaeal genomes have been sequenced, and [a large fraction o
 * Spacer self-matches to host genome
 * *cas* gene content and inferred CRISPR type
 
-We will collect these annotations in a database that can users can query through a GUI (graphical user interface). Neither of these exist yet, and we are looking for contributors! 
+We will collect these annotations in a database that can users can query through a GUI (graphical user interface). Neither of these exist yet, and we are looking for contributors!
```

Comment:
It looks like something with your text editor is changing line endings? I'm not sure why all these other paragraphs are showing up as edited.